torchrl/envs/vec_env.py

```diff
@@ -798,6 +799,19 @@ def to(self, device: DEVICE_TYPING):
         return self
 
 
+def recursively_strip_locks_from_state_dict(state_dict: OrderedDict) -> OrderedDict:
+    return OrderedDict(
```

Comment:
same: do we want to use OrderedDict?

Reply:
Same as above, existing code assumes state_dict is an OrderedDict, I have changed a few places in code that assumed this (in the tests) but there are others remaining, not sure if cleaning this up is the objective of this change, but I can do so. It might also make another good bootcamp task.